### ISSUE TITLE
[Keyboard Navigation - Panels]: Panel menu is not accessible when navigating between panels via keyboard

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66256]
### User Experience: 
 Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls, hence the user will fail to use the control's functionality. 

### Repro Steps:
Open #Url: https://play.grafana.org/   Sign in with valid credentials. "Home" page appears. Navigate to the "Traffic In/Out" graph and all its controls. Verify the issue.

 ### Actual Result: 
The panel menu is not accessible via keyboard i.e. on selecting tab key from "Logins" the focus goes into "Traffic In/Out" timeseries skipping the menu. This is due to the new menu only being visible on hover.

https://github.com/grafana/grafana/assets/74534993/27979fed-42e0-4471-9615-610b3c02d741

### Expected Result:
Panel menu should be accessible via keyboard.

### Severity
`SEV2`

**Comment by natellium** [comment]
This may have been fixed by the new panel header

**Comment by khushijain21** [comment]
I'd like to work on this - is it open?

**Comment by natellium** [comment]
Hey @khushijain21 , yes this is still open, feel free to pick it up :)
